prow/extra.yaml

```diff
@@ -0,0 +1,25 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: config-map-updater
+rules:
+- apiGroups: ["*"]
+  resources: ["configmaps"]
+  verbs: ["get", "list", "patch", "update"]
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: prow-config-bot
```

Comment:
one thing i tried to do (and probably didnt do a great job at!) was to document the secrets we're adding to the prow cluster as we add them at https://github.com/tektoncd/plumbing/tree/master/prow#creating-the-secrets <-- would be great to keep updating that and/or find a better way of keeping track

when i tried to create the prow cluster from scratch i found it pretty hard to work backward from the existing state of the cluster and understand why all the stuff in the cluster existed (or if it even needed to exist)

Reply:
OK, good idea, I'll fix that, thanks.

Reply:
I think I'll do this in tekton/README.md tough - following the same structure we have for the prow/README.md.

Reply:
Done